### DISCUSSION TITLE
Getting pending operations

### DISCUSSION THREAD

**Discussion by KieranFleckney** [Azure/azure-mobile-apps#906] [Q&A] [answered]
I was wondering if it was possible to get a list of all the pending operations? I know you can do client.PendingOperations to get the number operation pending. But it would be good to know which item are pending.

**Comment by adrianhall** [comment, marked as answer]
That's kind of hard, since you aren't attached to a specific table at that point (normally, you'd want to know what items are dirty so you can put an icon next to the item, for example).  You can use the ExecuteSql methods to query the operations queue, but it requires knowledge of the underlying structure of the database - which can change (but. reality, it doesn't change).

**Reply by KieranFleckney** [reply]
this was the exact reason I was looking to do it. I did think about doing the raw sql query but didn't know if I was missing something. But thank you answering it
